pennylane/ops/qubit/observables.py

```diff
@@ -161,6 +162,52 @@ def eigvals(self):
         """
         return self.eigendecomposition["eigval"]
 
+    @staticmethod
+    def compute_decomposition(A, wires):  # pylint: disable=arguments-differ
+        r"""Decomposes an observable as a sum of Pauli operators (static method).
+
+        Args:
+            A (array or Sequence): hermitian matrix
+            wires (Iterable[Any], Wires): wires that the operator acts on
+        Returns:
+            list[.PauliSentence]: decomposition of the hermitian matrix
+
+        **Examples**
+
+        >>> op = qml.X(0) + qml.Y(1) + 2 * qml.X(0) @ qml.Z(3)
+        >>> op_matrix = qml.matrix(op)
+        >>> qml.Hermitian.compute_decomposition(op_matrix, wires=['a', 'b', 'aux'])
+        [1.0 * Y(b)
+        + 1.0 * X(a)
+        + 2.0 * X(a) @ Z(aux)]
+        >>> op = np.array([[1, 1], [1, -1]]) / np.sqrt(2)
+        >>> qml.Hermitian.compute_decomposition(op, wires=0)
+        [0.7071067811865475 * X(0)
+        + 0.7071067811865475 * Z(0)]
+
+        """
+        A = qml.math.asarray(A)
+        A_SHAPE = qml.math.shape(A)
+        A_DIMENSION = A_SHAPE[0]
```

Comment:
Do we need all caps for these variables? I believe that's [recommended for module-level constants](https://peps.python.org/pep-0008/#constants).